index.js

```diff
@@ -9,12 +9,18 @@ var blocked = []
 var isBlocked = false
 
 function Plugin(
-	/* config.webpack */ webpackOptions,
-	/* config.webpackServer */ webpackServerOptions,
-	/* config.webpackMiddleware */ webpackMiddlewareOptions,
-	/* config.basePath */ basePath,
-	/* config.files */ files,
-	/* config.frameworks */ frameworks,
+	/* config.webpack */
+	webpackOptions,
```

Comment:
As @dignifiedquire stated, this does break the DI injector of karma. For karma-webpack the current thinking is jsbeautify is going to be more trouble that it's worth and will be removed shortly.